imagej/imagej-ops2-tutorial/docs/MigratingFromImageJOps.md

```diff
@@ -0,0 +1,198 @@
+# Migrating from ImageJ Ops
```

Comment:
Overall great document. I think it could benefit from the headings being numbered.

Reply:
Let me know what you think of 16554321

Reply:
Love it! This feels like a nice step-by-step migration document.